pkg/instrumentation/database.go

```diff
@@ -47,18 +47,18 @@ func newCallbacks(appName string) *callbacks {
 	}
 }
 
-func (c *callbacks) beforeCreate(scope *gorm.Scope)   { c.before(scope, "INSERT", c.serviceName) }
-func (c *callbacks) afterCreate(scope *gorm.Scope)    { c.after(scope) }
-func (c *callbacks) beforeQuery(scope *gorm.Scope)    { c.before(scope, "SELECT", c.serviceName) }
-func (c *callbacks) afterQuery(scope *gorm.Scope)     { c.after(scope) }
-func (c *callbacks) beforeUpdate(scope *gorm.Scope)   { c.before(scope, "UPDATE", c.serviceName) }
-func (c *callbacks) afterUpdate(scope *gorm.Scope)    { c.after(scope) }
-func (c *callbacks) beforeDelete(scope *gorm.Scope)   { c.before(scope, "DELETE", c.serviceName) }
-func (c *callbacks) afterDelete(scope *gorm.Scope)    { c.after(scope) }
-func (c *callbacks) beforeRowQuery(scope *gorm.Scope) { c.before(scope, "", c.serviceName) }
-func (c *callbacks) afterRowQuery(scope *gorm.Scope)  { c.after(scope) }
-func (c *callbacks) before(scope *gorm.Scope, operationName string, serviceName string) {
-	val, ok := scope.Get(ParentSpanGormKey)
+func (c *callbacks) beforeCreate(db *gorm.DB)   { c.before(db, "INSERT", c.serviceName) }
+func (c *callbacks) afterCreate(db *gorm.DB)    { c.after(db) }
+func (c *callbacks) beforeQuery(db *gorm.DB)    { c.before(db, "SELECT", c.serviceName) }
+func (c *callbacks) afterQuery(db *gorm.DB)     { c.after(db) }
+func (c *callbacks) beforeUpdate(db *gorm.DB)   { c.before(db, "UPDATE", c.serviceName) }
+func (c *callbacks) afterUpdate(db *gorm.DB)    { c.after(db) }
+func (c *callbacks) beforeDelete(db *gorm.DB)   { c.before(db, "DELETE", c.serviceName) }
+func (c *callbacks) afterDelete(db *gorm.DB)    { c.after(db) }
+func (c *callbacks) beforeRowQuery(db *gorm.DB) { c.before(db, "", c.serviceName) }
+func (c *callbacks) afterRowQuery(db *gorm.DB)  { c.after(db) }
```

Comment:
nit: Since we renamed `row_query` to `row`.

```suggestion
func (c *callbacks) beforeRow(db *gorm.DB) { c.before(db, "", c.serviceName) }
func (c *callbacks) afterRow(db *gorm.DB)  { c.after(db) }
```